Lib/test/test_cmd_line.py

```diff
@@ -453,6 +453,24 @@ def test_del___main__(self):
             print("del sys.modules['__main__']", file=script)
         assert_python_ok(filename)
 
+    def test_global_del_SystemExit(self):
```

Comment:
I don't think `test_cmd_line` is the right place for this. `test_gc` already has related tests (see e.g. `test_gc_main_module_at_shutdown()` there).